akka-cluster/src/main/scala/akka/cluster/ClusterDaemon.scala

```diff
@@ -1065,15 +1070,18 @@ private[cluster] class ClusterCoreDaemon(publisher: ActorRef) extends Actor with
     val localSeen = localOverview.seen
 
     val enoughMembers: Boolean = isMinNrOfMembersFulfilled
```

Comment:
Leaning towards only team members but not 100%. Maybe you would also want the team leader to postpone upping the team until the other DCs has started. Umm. Sounds a bit hard to reason about I to me.